root/treesitter.py

```diff
@@ -0,0 +1,48 @@
+from collections import Counter
+
+from tree_sitter import Language, Parser
+
+languages = {"python": Language("../build/my-languages.so", "python"),
+             "go": Language("../build/my-languages.so", "go"),
+             "javascript": Language("../build/my-languages.so", "javascript")}
+#             "java": Language("../build/my-languages.so", "java")}
+queries = {"python": Language("../build/my-languages.so", "python").query("""
+        (assignment left: (identifier) @var_name)
+        (class_definition name: (identifier) @class_name)
+        (function_definition name: (identifier) @func_name)
+       """),
+           "go": Language("../build/my-languages.so", "go"),
+           "javascript": Language("../build/my-languages.so", "javascript").query("""
+        (variable_declarator name: (identifier) @var_name)
+        (class_declaration name: (identifier) @class_name)
+        (function_declaration name: (identifier) @func_name)
+       """)}
+#     "java": Language("../build/my-languages.so", "java").query("""
+#  (
+#  variable_declarator name: (identifier) @var_name
+#  class_declaration name: (identifier) @class_name
+#  method_declaration name: (identifier) @func_name
+#  )
+# """)}
+parser = Parser()
```

Comment:
please check if it's a correct way to make singletone
https://webdevblog.ru/realizaciya-shablona-singleton-v-python/